Activities/FTP/UiPath.FTP/FtpObjectType.cs

```diff
@@ -1,6 +1,8 @@
 ﻿namespace UiPath.FTP
 {
-    public enum FtpObjectType
+    //Prefix the name with uipath due to conflict with FtpObjectType from FluentFTP
+    //see https://github.com/robinrodricks/FluentFTP/blob/8e7ba0ad892759c40b799748241bc4d20d8150f1/FluentFTP/Enums/FtpObjectType.cs#L7
+    public enum UiPathFtpObjectType
```

Comment:
could we make it `internal`?

Reply:
yes

Reply:
I double checked, we cannot make it internal since it's part of the the view model for the activity that enumerates the files
https://github.com/UiPath/Community.Activities/blob/4818fc6e3d22976fe6f1d4ffc72b03941e8b50c5/Activities/FTP/UiPath.FTP.Activities/NetCore/ViewModels/EnumerateObjectsViewModel.cs#L42
I will prefix with the namespace name when necessary to avoid conflicts